packages/shared/src/components/cards/squad/SquadsDirectoryFeed.tsx

```diff
@@ -1,4 +1,6 @@
 import React, { ReactElement, ReactNode } from 'react';
+import Link from 'next/link';
```

Comment:
This is the wrong one right we need the custom one since next 14

Reply:
🤦‍♀️